src/dto/index.ts

```diff
@@ -7,5 +7,11 @@ export interface ILyricsAndDetails {
   lyrics: string;
   title: string;
   artist: string;
-  geniusUrl: string;
+  geniusUrl?: string;
+  syncedLyricsArray?: Array<ILRCContent>;
 }
+
+export interface ILRCContent {
+  content: string;
+  timestamp: number;
+}
```

Comment:
You have this in `client\src\types\trackLyrics.ts` - can reference that here and drop this